src/document.ts

```diff
@@ -5,15 +5,18 @@ type DictionaryMetadataValue = Record<string, BaseMetadataValue>;
 export type MetadataValue = BaseMetadataValue | DictionaryMetadataValue;
 export type Metadata = Record<string, MetadataValue>;
 
+export enum CompressionTypeEnum {
```

Comment:
For internal use with Bueno validation `EnumValue`.
This is not exported publicly.